nengo/utils/graphs.py

```diff
@@ -98,6 +98,43 @@ def toposort(edges):
     return ordered
 
 
+def transitive_closure(edges, topo_sorted=None):
+    """Constructs the transitive_closure of a DAG.
```

Comment:
Not sure, it wouldn't be correct to say the transititve closure is a “list of adjancencies” (that is only the data structure used to represent graphs and even that is only partially correct because we're using sets). To define it correctly and precisely it would probably require quite a bit of text. I would prefer to keep it concise because it is a well-defined term that can easily be looked up.